src/lib/Microsoft.Health.Fhir.Ingest/Telemetry/NormalizationExceptionTelemetryProcessor.cs

```diff
@@ -14,9 +15,10 @@ namespace Microsoft.Health.Fhir.Ingest.Telemetry
     public class NormalizationExceptionTelemetryProcessor : ExceptionTelemetryProcessor
     {
         private readonly string _connectorStage = ConnectorOperation.Normalization;
+        private static readonly Type[] DefaultExceptions = new[] { typeof(IncompatibleDataException) };
 
-        public NormalizationExceptionTelemetryProcessor()
-            : base(typeof(IncompatibleDataException))
+        public NormalizationExceptionTelemetryProcessor(params Type[] handledExceptionTypes)
+            : base(handledExceptionTypes.Union(DefaultExceptions).ToArray())
```

Comment:
Do we need a `null` check on `handledExceptionTypes`?

Reply:
Nope, tested the Union on a null enumeration works with an array of one with the DefaultExceptions content